### PR TITLE
Created N을/를 위해(서)

### PR DESCRIPTION
**Pull request by Kikkah070**
The example sentences are from this [source](https://www.koreantopik.com/2018/11/l1g52-n-v-grammar-for-sake-of-in-order.html)

### CODE REVIEW

File: point/을_위해서.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+definitions:
+  - slug: before-a-certain-time
+    name: Before a certain time
+    english_alternatives: For the sake of, in order to
+    meaning:
+    examples:
+      - sentence: 군인은 <f>나라를 위해서</f> 일하는 사람이에요.
+        type: simple
+        translated: Soldiers are people who work for their country.
+        audio_url:
```
Comment (Alaanor) [inline point/을_위해서.yaml:11]:
```suggestion
        audio_url: https://r2.kimchi-reader.app/grammar/군인은_나라를_위해서_일하는_사람이에요__2024-05-27.mp3
```

File: point/을_위해서.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      - sentence: <f>건강을 위해서</f> 매일 비타민을 먹고 있어요
```
Comment (Alaanor) [inline point/을_위해서.yaml:12]:
```suggestion
      - sentence: <f>건강을 위해서</f> 매일 비타민을 먹고 있어요.
```

File: point/을_위해서.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        type: simple
+        translated: I take vitamins daily for my health.
+        audio_url:
```
Comment (Alaanor) [inline point/을_위해서.yaml:15]:
```suggestion
        audio_url: https://r2.kimchi-reader.app/grammar/건강을_위해서_매일_비타민을_먹고_있어요__2024-05-27.mp3
```

File: point/을_위해서.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+metadata:
+  type: composite
+details: |-
+  # Usage with verbs {#usage-with-verbs}
+
+  Can also be used with verbs in the form of <f>기 위해(서)</f>
+
+  ::example[시험에 <f>합격하기 위해서</f> 열심히 공부했어요 | I studied hard in order to pass the test| ]
```
Comment (Alaanor) [inline point/을_위해서.yaml:23]:
```suggestion
  ::example[시험에 <f>합격하기 위해서</f> 열심히 공부했어요. | I studied hard in order to pass the test| https://r2.kimchi-reader.app/grammar/시험에_합격하기_위해서_열심히_공부했어요__2024-05-27.mp3]
```

File: point/을_위해서.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  ::example[스마트폰을 <f>사기 위해서</f> 돈을 모았어요. | I saved up money to buy a smartphone.  | ]
```
Comment (Alaanor) [inline point/을_위해서.yaml:24]:
```suggestion
  ::example[스마트폰을 <f>사기 위해서</f> 돈을 모았어요. | I saved up money to buy a smartphone.  | https://r2.kimchi-reader.app/grammar/스마트폰을_사기_위해서_돈을_모았어요__2024-05-27.mp3]
```

File: point/을_위해서.yaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+name: 을/를 위해(서)
+definitions:
+  - slug: before-a-certain-time
+    name: Before a certain time
+    english_alternatives: For the sake of, in order to
+    meaning:
```
Comment (Alaanor) [inline point/을_위해서.yaml:6]: missing meaning

**Review by Alaanor — CHANGES_REQUESTED** [review]

File: point/을_위해서.yaml (hunk 6 above)
Comment (Kikkah070) [reply point/을_위해서.yaml]: Added the meaning

**Pull request merged**